doc/sphinx/source/conformance.rst

```diff
@@ -0,0 +1,186 @@
+.. _pocl-conformance:
+
+=======================
+Pocl OpenCL conformance
+=======================
+
+Conformance related CMake options
+---------------------------------
+
+- ``-DENABLE_CONFORMANCE=ON/OFF``
+  This is mostly related to the kernel library (the runtime is always conformant
+  on x86). Defaults to ON.
+  Non-conformant kernel library might be somewhat faster, at the expense of
+  precision and/or range. Note that conformance was tested **only** on certain
+  hardware and software (Linux, x86-64, CPU with AVX & FMA instructions).
+
+How to run the conformance test suite on your hardware
+------------------------------------------------------
+
+To run the shortened version, run: ``ctest -L conformance_suite_short``
+This might take a few hours on slow hardware.
+
+To run the full conformance testsuite, run: ``ctest -L conformance_suite``
+Note that this can take a week to finish on slow hardware, and about a day
+on fast hardware (6C/12T Intel or equivalent).
+
+Known issues with the conformance testsuite
+-------------------------------------------
+
+- the "not" operator test (``math_brute_force/bruteforce not``) may fail to
+  compile with LLVM 4.0 with certain vector sizes on some hardware.
+  This does not seem to affect the rest of the testsuite in any way, and
+  appears to be fixed with LLVM 5.0
+
+- a few tests from ``basic/test_basic`` may fail / segfault because they
+  request a huge amount of memory for buffers.
+
+.. _sigfpe-handler:
```

Comment:
I think this anchor is in a wrong place, should be just before the issue item?

Reply:
it should, and was there originally, but sphinx screams and refuses to make html :[